README.md

```diff
@@ -1,285 +1,51 @@
-<a name="readme-top"></a>
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
+# Linters Set-Up
 
-Modify this file to match your project and remove sections that don't apply.
+> Linters are used for keeping a code that is in sync with best code practices
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
 
-After you're finished please remove all the comments and instructions!
--->
+## Built With
 
-<div align="center">
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
kindly, add the languages you used to make your repository descriptive from the readme for others.